zebra-chain/src/orchard/note/ciphertexts.rs

```diff
@@ -11,7 +11,7 @@ use crate::serialization::{SerializationError, ZcashDeserialize, ZcashSerialize}
 pub struct EncryptedNote(#[serde(with = "BigArray")] pub(crate) [u8; 580]);
 
 // These impls all only exist because of array length restrictions.
-// TODO: use const generics https://github.com/ZcashFoundation/zebra/issues/2042
+// TODO: use const generics <https://github.com/ZcashFoundation/zebra/issues/2042>
```

Comment:
Optional: not needed.
```suggestion
// TODO: use const generics https://github.com/ZcashFoundation/zebra/issues/2042
```

Reply:
Resolved in [36fc845](https://github.com/ZcashFoundation/zebra/pull/4494/commits/36fc8453d4239291deb6e3f692a0d6b9253f17e8)